### DISCUSSION TITLE
How to enable the @ variable in Chat

### DISCUSSION THREAD

**Discussion by SkyKingPX** [EssentialsX/Essentials#5255] [Q&A] [answered]
How to enable the @ variable in Chat and in command blocks?
(Like: /kill -entity-)
Can I do this?

**Comment by Evidentsinger14** [comment, marked as answer]
use `/minecraft:kill` in your command block, or add `kill` to `command-block-overrides` in `commands.yml` (located in the root server folder).

**Reply by SkyKingPX** [reply]
When I do this in the commands.yml, can I use it also in the Chat?

**Reply by Evidentsinger14** [reply]
If you use command-block-overrides, no. you would still have to run it as `/minecraft:kill` - if you want to completely disable the Essentials kill command, you can do so in `disabled-commands` in the config.yml.

**Reply by SkyKingPX** [reply]
Thanks 🙃
